LongoMatch.DB/LongoMatch.DB.Net45.csproj

```
@@ -66,8 +64,10 @@
   <Import Project="..\.nuget\NuGet.targets" />
   <Import Project="$(SolutionDir)\packages\SQLitePCL.raw.0.8.6\build\net45\SQLitePCL.raw.targets" Condition="Exists('$(SolutionDir)\packages\SQLitePCL.raw.0.8.6\build\net45\SQLitePCL.raw.targets')" />
   <Import Project="..\packages\SQLitePCL.native.sqlite3.v110_xp.0.9.1\build\SQLitePCL.native.sqlite3.v110_xp.targets" Condition="Exists('..\packages\SQLitePCL.native.sqlite3.v110_xp.0.9.1\build\SQLitePCL.native.sqlite3.v110_xp.targets')" />
+  <Import Project="..\packages\Fody.1.29.4\build\portable-net+sl+win+wpa+wp\Fody.targets" Condition="Exists('..\packages\Fody.1.29.4\build\portable-net+sl+win+wpa+wp\Fody.targets')" />
```

Use $(SolutionDir), also don't make it conditional and let the build fail if not found instead of just skeeping the weaving process.